app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -749,6 +751,27 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         client.track(CTA_CLICKED.eventName, props)
     }
 
+    /**
+     * Tracks a discover project clicks on the search result.
```

Comment:
Please add parameter documentation to this

Reply:
Done ✅